soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -1258,331 +1260,239 @@ static bool GenerateRandoHandler(std::shared_ptr<LUS::Console> Console, const st
-        return CMD_FAILED;
+        ERR_MESSAGE("[SOH] seed|count value must be a number.");
+        return 1;
     }
 
 
-    LUS::GetConsole()->SendErrorMessage("[SOH] Rando generation already in progress");
-    return CMD_FAILED;
+    ERR_MESSAGE("[SOH] Rando generation already in progress");
+    return 1;
 }
 
-static bool CosmeticsHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args) {
+static bool CosmeticsHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args, std::string* output) {
     if (args.size() < 2) {
-        LUS::GetConsole()->SendErrorMessage("[SOH] Unexpected arguments passed");
-        return CMD_FAILED;
+        ERR_MESSAGE("[SOH] Unexpected arguments passed");
+        return 1;
     }
 
     if (args[1].compare("reset") == 0) {
         CosmeticsEditor_ResetAll();
     } else if (args[1].compare("randomize") == 0) {
         CosmeticsEditor_RandomizeAll();
     } else {
-        LUS::GetConsole()->SendErrorMessage("[SOH] Invalid argument passed, must be 'reset' or 'randomize'");
-        return CMD_FAILED;
+        ERR_MESSAGE("[SOH] Invalid argument passed, must be 'reset' or 'randomize'");
+        return 1;
     }
 
-    return CMD_SUCCESS;
+    return 0;
 }
 
-static bool SfxHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args) {
+static bool SfxHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args, std::string* output) {
     if (args.size() < 2) {
-        LUS::GetConsole()->SendErrorMessage("[SOH] Unexpected arguments passed");
-        return CMD_FAILED;
+        ERR_MESSAGE("[SOH] Unexpected arguments passed");
+        return 1;
     }
 
     if (args[1].compare("reset") == 0) {
         AudioEditor_ResetAll();
     } else if (args[1].compare("randomize") == 0) {
         AudioEditor_RandomizeAll();
     } else {
-        LUS::GetConsole()->SendErrorMessage("[SOH] Invalid argument passed, must be 'reset' or 'randomize'");
-        return CMD_FAILED;
+        ERR_MESSAGE("[SOH] Invalid argument passed, must be 'reset' or 'randomize'");
+        return 1;
     }
 
-    return CMD_SUCCESS;
-}
-
-#define VARTYPE_INTEGER 0
-#define VARTYPE_FLOAT   1
-#define VARTYPE_STRING  2
-#define VARTYPE_RGBA    3
-
-static int CheckVarType(const std::string& input)
-{
-    int result = VARTYPE_STRING;
-
-    if (input[0] == '#') {
-        return VARTYPE_RGBA;
-    }
-
-    for (size_t i = 0; i < input.size(); i++)
-    {
-        if (!(std::isdigit(input[i]) || input[i] == '.'))
-        {
-            break;
-        }
-        else
-        {
-            if (input[i] == '.')
-                result = VARTYPE_FLOAT;
-            else if (std::isdigit(input[i]) && result != VARTYPE_FLOAT)
-                result = VARTYPE_INTEGER;
-        }
-    }
-
-    return result;
-}
-
-static bool SetCVarHandler(std::shared_ptr<LUS::Console> Console, const std::vector<std::string>& args) {
```

Comment:
I realize this command was kinda gross, but it was useful, are we sure we want to remove?

Reply:
it lives in LUS now https://github.com/Kenix3/libultraship/blob/6d29ff2dbc4507b59771d5af84cb58eaacd2f9a7/src/window/gui/ConsoleWindow.cpp#L121-L152